templates/blog/topics/juju.html

```diff
@@ -6,27 +6,15 @@
 
 {% block content %} 
 
-  <section class="p-strip--accent p-blog-topic__strip is-dark is-deep u-image-position" style="overflow: hidden; background-color: #333">
-    <div class="row header-strip u-equal-height">
-      <div class="col-6">
-        <div class="p-heading-icon">
-          <div class="p-heading-icon__header">
-            <img class="p-blog-heading__icon" src="https://assets.ubuntu.com/v1/67057b2c-product-page-juju-logo.svg" alt="" style="margin-bottom: .75rem;"/>
-          </div>
-          <p class="p-heading--three">Operate big software at scale on any cloud</p>
-          <p>All the latest news on Juju – the open source application modelling tool for public and private clouds.</p>
-        </div>
-        <p>
-          <a class="p-link--external p-link--inverted" href="https://jujucharms.com">Learn more about Juju</a>
-        </p>
-      </div>
-      <div class="col-5 col-start-large-8">
-        <img
-          src="https://assets.ubuntu.com/v1/e8b86f15-Juju-Visual-Screen.png?w=576"
-          class="u-hide--small u-image-position--bottom"
-          style="padding-top: 2.5rem; max-height: 100%;"
-          alt="" />
-      </div>
+  <section class="p-strip--suru-blog-header">
+    <div class="u-fixed-width">
+      <p>
+        <img class="p-blog-heading__icon" src="https://assets.ubuntu.com/v1/67057b2c-product-page-juju-logo.svg" alt="" width="140" height="50" style="height: 50px;"/>
+      </p>
+      <h2 class="p-heading--four">All the latest news on Juju &ndash; the open source application<br class="u-hide u-show--large" />modelling tool for public and private clouds.</h2>
+      <p>
+        <a class="p-link--external p-link--inverted" href="https://jujucharms.com">Learn more about Juju</a>
```

Comment:
I think this should go to jaas.ai now
```suggestion
        <a class="p-link--external p-link--inverted" href="https://jaas.ai">Learn more about Juju</a>
```